### PR TITLE
Create geany.xml

### PR DESCRIPTION
**Pull request by pas-calc**
Geany recent files cleaner added

### CODE REVIEW

File: pending/xed-editor.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+<cleaner id="xed">
```
Comment (az0) [inline pending/xed-editor.xml:24]: Please add `os="linux"` or Unix on the cleaner element

**Comment by az0** [comment]
Does dconf work like this? Based on your comment in https://github.com/bleachbit/wishlist/issues/97 , it seems it doesn't work

**Comment by pas-calc** [comment]
> Does dconf work like this? Based on your comment in [bleachbit/wishlist#97](https://github.com/bleachbit/wishlist/issues/97) , it seems it doesn't work

Sorry, yes you are right. I am a bit new to github, so what I want to do is:

- Merge geany.xml (this cleaner works fine) - could you do this on the single file?

- the xed cleaner does not work yet (as described in the wishlist link you mentioned)

**Comment by az0** [comment]
I merged geany and made a few tweaks. Thanks

**Pull request merged**